pkg/api/service.go

```diff
@@ -95,7 +96,7 @@ func (a *apiServer) handleEvent(ctx context.Context, e *event.Event) (string, er
 
 	l.Debug().Str("event", e.Name).Msg("handling event")
 
-	if strings.HasPrefix(strings.ToLower(e.Name), "inngest/") {
+	if strings.HasPrefix(strings.ToLower(e.Name), "inngest/") && e.Name != consts.InvokeEventName {
```

Comment:
This makes sense, though breaks the easy contract of "_you can never send `inngest/*` events_."

My initial thought was to send which functions should be invoked in the event itself in addition to the `name`, `data`, and `user` properties, but this obviously wouldn't work for crons.

What's the shape of the `inngest/function.invoked` event?

Reply:
You can post any data to the invoke endpoint as a JSON object, which we copy into `event.data`.  We add a specific `_inngest_fn` field which stores the slug of the function to call.

We could block the regular event ingestion endpoint from receiving any `inngest/*` functions.  That means users can never send random `inngest/*` events _at all_ - the only way to generate the `inngest/function.invoked` event is via the invoke API

Reply:
Done!